app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -349,8 +349,29 @@ def parse_container(container, pod_id)
         :backing_ref     => container.containerID,
         :container_image => parse_container_image(container.image, container.imageID)
       }
-
-      # TODO, state
+      Hash(container.state).each do |key, val|
+        h.merge!(
+          :state       => key,
+          :reason      => val['reason'],
+          :started_at  => val['startedAt'],
+          :finished_at => val['finishedAt'],
```

Comment:
@moolitayer datetimes should be explicitly converted here (not implicitly by activerecord or the db). (Right? here `finishedAt` and all the others are strings).